apisix/core/ctx.lua

```diff
@@ -274,6 +274,15 @@ do
                     end
                 end
 
+            elseif core_str.has_prefix(key, "uri_arg_") then
```

Comment:
The built-in variable only works when the router engine is `radixtree_uri_with_parameter`, we should add comment about this

Reply:
Partially correct, as `curr_req_matched` is set in https://github.com/apache/apisix/blob/538b964c3bb5a60d8b6016aa0b552439c6307f1b/apisix/http/router/radixtree_host_uri.lua#L83 and based on https://github.com/apache/apisix/blob/538b964c3bb5a60d8b6016aa0b552439c6307f1b/apisix/http/router/radixtree_host_uri.lua#L179 it seems `_host` is even valid without `radixtree_uri_with_parameter` also looking at [`lib/resty/radixtree.lua`](https://github.com/api7/lua-resty-radixtree/blob/master/lib/resty/radixtree.lua) it seems `_action`, `_path` and `_host` can be available.

Reply:
But your title is `Provide uri_arg_ when radixtree_uri_with_parameter`, and you need to tell the users what they can access

Reply:
After some testing it's seems only `_path` is consistently available so I added a comment. 
Is this good enough?